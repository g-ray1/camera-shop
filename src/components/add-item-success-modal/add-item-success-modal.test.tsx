import { render } from '@testing-library/react';
import AddItemSuccessModal from './add-item-success-modal';
import { BrowserRouter } from 'react-router-dom';

jest.mock('react-redux');

describe('AddItemSuccessModal component', () => {
  it('Snapshot', () => {
    const view = render(<AddItemSuccessModal />, {wrapper: BrowserRouter});

    expect(view).toMatchSnapshot();
  });
});
