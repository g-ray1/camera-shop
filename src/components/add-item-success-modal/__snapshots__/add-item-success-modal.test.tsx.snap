// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddItemSuccessModal component Snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <p
        class="title title--h4"
      >
        Товар успешно добавлен в корзину
      </p>
      <svg
        aria-hidden="true"
        class="modal__icon"
        height="80"
        width="86"
      >
        <use
          xlink:href="#icon-success"
        />
      </svg>
      <div
        class="modal__buttons"
      >
        <button
          class="btn btn--transparent modal__btn"
        >
          Продолжить покупки
        </button>
        <button
          class="btn btn--purple modal__btn modal__btn--fit-width"
        >
          Перейти в корзину
        </button>
      </div>
    </div>
  </body>,
  "container": <div>
    <p
      class="title title--h4"
    >
      Товар успешно добавлен в корзину
    </p>
    <svg
      aria-hidden="true"
      class="modal__icon"
      height="80"
      width="86"
    >
      <use
        xlink:href="#icon-success"
      />
    </svg>
    <div
      class="modal__buttons"
    >
      <button
        class="btn btn--transparent modal__btn"
      >
        Продолжить покупки
      </button>
      <button
        class="btn btn--purple modal__btn modal__btn--fit-width"
      >
        Перейти в корзину
      </button>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
